package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polkadot-deployer",
-  "version": "2.8.1",
+  "version": "2.8.2",
```

Comment:
or 2.9.0 ?

Reply:
I would go for 2.9.0 given that we introduce new features.